code/src/App.js

```diff
@@ -1,9 +1,89 @@
-import React from 'react'
+import React, { useState } from 'react'
+import './app.css'
+import './index.css'
+
+
+const ageGroup = ['too young', '19-35', '35-65', '+65'];
+const option0 = ''
+const option1 = 'Candlelight dinner in a cosy restaurant'
+const option2 = 'Netlfix and Chill'
+const option3 = 'Dancing on my own'
+const option4 = 'Party with my friends'
+const option5 = 'Headbangers ball concert'
```

Comment:
Maybe this would suit as an object or array instead?

Reply:
Thanks for the review! I wanted to do something with this but was unsure how to make it in another way, would you please give me a hint on how to change this?.